src/by.jl

```diff
@@ -1,8 +1,9 @@
 """
     by(formula, by, design, function, params)
 
-Generate subsets of a survey design.
+Estimate the population parameters of for subpopulations of interest for a simple random sample.  
```

Comment:
Remove "of".

Reply:
The description should be the same for all methods, for all designs. In the API, there only needs to be a general description. Only if there's a very special method that is not intuitive to use after you see the main method, only then that method should be added (and described) separately. In this case, we should only keep the description here, something like:
"Estimate the population parameters for subpopulations of interest."

Reply:
`by` will be removed in the next PR, so will be marking this as resolved in this PR.